enrollment-server-onboarding-domain-model/src/main/java/com/wultra/app/enrollmentserver/model/enumeration/ErrorOrigin.java

```diff
@@ -32,9 +32,13 @@ public enum ErrorOrigin {
 
     OTP_VERIFICATION,
 
+    USER_VERIFICATION,
```

Comment:
When these two new error origins approved, I will change the documentation.

Reply:
Ok.